Makefile

```diff
@@ -21,23 +21,27 @@ all: pkg tar
 pkg: deb rpm
 
 deb: populate-deb
+	(cd pkg/deb/; \
+		DEBMAIL="stackdriver-agents@google.com" DEBFULLNAME="Stackdriver Agents" \
```

Comment:
These variables are not needed anymore -- just give the `--controlmaint` parameter to `dch`.

Reply:
Done.